skipper.env

```diff
@@ -89,12 +91,19 @@ DEBUG_SERVICE_PORT
 SSH_CLIENT
 LOCAL_ASSISTED_ORG
 VSPHERE_CLUSTER
+vsphere_cluster
 VSPHERE_USERNAME
+vsphere_username
 VSPHERE_NETWORK
+vsphere_network
 VSPHERE_VCENTER
+vsphere_vcenter
 VSPHERE_DATACENTER
+vsphere_datacenter
 VSPHERE_DATASTORE
+vsphere_datastore
 VSPHERE_PASSWORD
+vsphere_password
```

Comment:
Why in lower case and why it's duplicated?

Reply:
The CI jobs use lowercase parameters while test-infra uses uppercase. This makes both of them work.